### PR TITLE
- Added a metadata attribute 'ssd_count' to spark_service.

### PR DESCRIPTION
**Pull request by asaksena**

### CODE REVIEW

**Comment by gareth-ferneyhough** [comment]
@asaksena: can you fix your linter errors? They are causing Travis CI to fail. Run hooks/check-everything to see details.

**Comment by asaksena** [comment]
@gareth-ferneyhough I have fixed the linter errors

File: perfkitbenchmarker/spark_service.py [hunk 1]
```diff
@@ -151,6 +151,11 @@ def GetMetadata(self):
       basic_data.update({'num_workers': str(self.spec.worker_group.vm_count),
                          'worker_machine_type':
                          str(self.spec.worker_group.vm_spec.machine_type)})
+
+    if self.spec.worker_group.vm_spec.num_local_ssds is not None and \
```
Comment (asaksena) [inline perfkitbenchmarker/spark_service.py]: @ehankland Now, dataproc overrides the getmetadata method and evaluates the ssd attributes. Will have to look into how this is handled for PKB managed clusters in a later PR. Let me know if this is better

File: perfkitbenchmarker/providers/gcp/gcp_dataproc.py [hunk 2]
```diff
@@ -174,3 +174,11 @@ def SubmitJob(self, jarfile, classname, job_poll_interval=None,
 
   def SetClusterProperty(self):
     pass
+
+  def GetMetadata(self):
+    basic_data = super(GcpDataproc, self).GetMetadata()
+    if (self.spec.worker_group.vm_spec.num_local_ssds is not None and
```
Comment (asaksena) [inline perfkitbenchmarker/providers/gcp/gcp_dataproc.py]: Thanks for the tip. Done.

**Pull request merged**